modules/ROOT/pages/proc_defining-and-creating-pipelineresources.adoc

```diff
@@ -10,53 +10,83 @@
 [discrete]
 === Procedure
 
-After you create `Task` s, create `PipelineResources` that contain the specifics of the Git repository and the image registry to be used in the `Pipeline` during execution as follows:
+After you create `Task`s, create `PipelineResources` that contain the specifics of the Git repository and the image registry to be used in the `Pipeline` during execution as follows:
 
-. Create a `PipelineResource` YAML file with a `PipelineResource` that defines the Git repository and reference for the PetClinic application as follows:
+. Create a `resources.yaml` file and inside of it create `PipelineResource`s which contain the specifics of the git repository and image registry to be used in the pipeline during execution.
```

Comment:
Create a `resources.yaml` file with the `PipelineResource`s that contain the specifics of the Git repository and the image registry to be used in the pipeline during execution.
git should be Git in all instances